interface/backend/static/views.py

```diff
@@ -2,5 +2,21 @@
 from django.shortcuts import render
 
 
-def home(request):
-    return render(request, '../frontend/index.html', {'short_hash': settings.APP_VERSION_NUMBER})
+def get_version_context():
```

Comment:
This should be moved to a Django "context processor" so that we don't have to include it in every single request.

In fact, what I suggest you do is add a ``settings_context`` processor that injects the ``django.conf.settings`` variable. This will mean we just have to do ``{{ settings.APP_VERSION_NUMBER}}`` in our template.

Reply:
That definitely makes sense, thanks! Are there any opinions on favoring libraries over self-implemented stuff? I mean there is [django-useful](http://django-useful.readthedocs.io/en/latest/usage.html#context-processors), which offers `useful.context_processors.settings` and which does exactly what we want - but one could also [build that on your own](https://stackoverflow.com/a/433209) which adds some more hoops but no further dependency. What would you favor?

Reply:
I feel strongly that this should not be tackled. Despite the validity at present of getting the layout mocked out while we wait for Vue.js usage to take over, the use of Django templates should be aggressively minimized.

Providing the short hash or something like it should be done via API, and probably won't even be its own view but rather as a piece of metadata in some general request(s) if not every response.

Reply:
@lamby Now that the initial bootstrap template has been merged, I squashed the commits together with the commits to update the template paths and to add a context processor.
@isms Does it mean that this issue/PR is completely blocked because we're waiting for the Vue.js stuff? Do you have any suggestions which call should deliver the version?